src/platform/site-wide/alerts/MobileAppCallout.jsx

```diff
@@ -36,10 +36,9 @@ const MobileAppCallout = ({
                 rel="noreferrer noopener"
```

Comment:
I don't think Public Websites owns this. Its only usage is in the src/applications/personalization folder